docs/fornax_test/830_release_testplan.md

```diff
@@ -0,0 +1,584 @@
+```
+Verify that the deployment is created in Cluster D only.
+
+Step 5: In the root operator machine, run command:
+```
+kubectl get mission 
+```
+Verify that the mission states of cluster B and B/C are reported as "not match", while the state of cluster B/C/D is the status info of the deployment.
+
+
+**Test Case 14: The information in the edge cluster status**
+
+Step 1: Run Command in the root operator machine
+```
+kubectl get edgeclusters
+```
+
+Verify the following information of edge cluster B are correctly displayed:
+
+a. LastHeartBeat 
+
+b. HealthStatus: should be "healthy"
+
+c. SubEdgeClusters: should include "C" and "C/D"
+
+d. Received_Missions
+
+e. Matched_Missions
+
+Step 2: Run Command in the root operator machine
+```
+kubectl get edgeclusters --kubeconfig=[Cluster_B_kubeconfig]
+```
+
+Verify the following information of edge cluster C are correctly displayed:
+
+a. LastHeartBeat 
+
+b. HealthStatus: should be "healthy"
+
+c. SubEdgeClusters: should include "D""
+
+d. Received_Missions
+
+e. Matched_Missions
+
+**Test Case 15: An edge cluster picks up the new mission added during its disconnection when reconnected to the cloud**
+
+Continuing from the previous test case, do the following:
+
+Step 1: delete all the missions deployed using command 
+```
+kubectl delete mission [mission_name]
+```
+
+And double check they are all gone using command
+```
+kubectl get missions
+```
+
+Step 2: kill the process of edgecore in cluster B. 
+
+Step 3: Run the following command and verify it returns successfully
+```
+kubectl apply -f tests/edgecluster/data/missions/deployment-to-all.yaml
+```
+
+Step 4: run command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get missions --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the mission is **NOT** created in the edge clusters B, C and D. 
+
+
+Step 5: run command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get deployment --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the deployment specified in the mission content is **NOT** created in the edge clusters B, C and D. 
+
+Step 6: restart the edgecore in edge-cluster mode and wait 20 seconds
+```
+edgecore --edgecluster
+```
+
+Step 7:  run command, with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get missions --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the mission is created in the edge clusters B, C and D.  
+
+Step 8:  run command , with [edge_cluster_kubeconfig] set to the kubeconfig files of cluster B, C and D respectively,
+```
+kubectl get deployment --kubeconfig=[edge_cluster_kubeconfig]
+```
+Verify that the deployment specified in the mission content is created in the edge clusters B, C and D.  
+
+**Test Case 16: An edge cluster updates the mission updated during its disconnection when reconnected to the cloud**
```

Comment:
"An edge cluster updates the mission updated during its disconnection when reconnected to the cloud"

-->

An edge cluster updates the mission during its disconnection from the cloud

This line reads a bit confusing to me, so the suggested change above may be wrong.

Reply:
Rewritten as " An edge cluster picks up the change when reconnected if the mission is updated during its disconnection"